app/utils/callback/lava.py

```diff
@@ -238,14 +272,7 @@ def _add_test_log(meta, job_log, base_path, suite):
     """
     log = yaml.load(job_log, Loader=yaml.CLoader)
 
-    dir_path = os.path.join(
-        base_path,
-        meta[models.JOB_KEY],
-        meta[models.GIT_BRANCH_KEY],
-        meta[models.KERNEL_KEY],
-        meta[models.ARCHITECTURE_KEY],
-        meta[models.DEFCONFIG_FULL_KEY],
-        meta[models.LAB_NAME_KEY])
+    dir_path = meta['dir_path']
```

Comment:
The convention in this code base is to use double quotes everywhere for strings, so `meta["dir_path"]`.